packages/binding-mqtt/test/mqtt-client-subscribe-test.unit.ts

```diff
@@ -75,8 +75,43 @@ describe("MQTT client implementation", () => {
                         done(err);
                     }
                 })
-                .then(() => mqttClient.invokeResource(form, new Content("", Readable.from(Buffer.from("test")))))
-                .then(() => mqttClient.stop())
+                .then(async (sub) => {
+                    await mqttClient.invokeResource(form, new Content("", Readable.from(Buffer.from("test"))));
+                    // Need to manually unsubscribe because stopping the client will unsubscribe all subscriptions
```

Comment:
```suggestion
                    // Need to manually unsubscribe because stopping the client will not unsubscribe all subscriptions
```

Forgot to put a **not**